summoner-cli/src/Summoner/Template/Cabal.hs

```diff
@@ -80,139 +75,125 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
         settingsTestedVersions
 
     sourceRepository :: Text
-    sourceRepository =
-        [text|
-        $endLine
-        source-repository head
-          type:                git
-          location:            ${githubUrl}.git
-        |]
+    sourceRepository = unlines
+        [ ""
+        , "source-repository head"
+        , "  type:                git"
+        , "  location:            " <> githubUrl <> ".git"
+        ]
 
     commonStanza :: Text
-    commonStanza =
-        [text|
-        $endLine
-        common common-options
-          build-depends:       base $baseBounds
-          $customPrelude
-          $ghcOptions
-
-          default-language:    Haskell2010
-    |] <> defaultExtensions
+    commonStanza = unlines $
+        [ ""
+        , "common common-options"
+        , "  build-depends:       base " <> baseBounds
+        ]
+        <> customPrelude
+        <> ("" : map ("  " <>) ghcOptions)
+        <>
+        ( ""
+        : "  default-language:    Haskell2010"
+        : defaultExtensions
+        )
 
     baseBounds :: Text
     baseBounds = cabalBaseVersions settingsTestedVersions
 
-    ghcOptions :: Text
+    ghcOptions :: [Text]
     ghcOptions = case settingsGhcOptions of
-        [] -> defaultGhcOptions
-        x:xs ->
-            let customGhcOptions = T.intercalate "\n" $ x : map (T.replicate 21 " " <>) xs in
-            [text|
-            ghc-options:         $customGhcOptions
-            |]
+        []   -> defaultGhcOptions
+        x:xs -> "ghc-options:         " <> x : map (T.replicate 21 " " <>) xs
```

Comment:
Maybe we can use `indent 21` from `colourista` here? Also, `indent` is implemented using `sconcat` :slightly_smiling_face:

Reply:
Oh yeah, this will be so much better! 👍